src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -101,12 +113,18 @@ void *frontendservicesesion(void *dummyPt) {
     PerformanceSQLiteDBInterface *perfSqlite = sessionargs->perfSqlite;
     JobScheduler *jobScheduler = sessionargs->jobScheduler;
     delete sessionargs;
+    if (currentFESession++ > Conts::MAX_FE_SESSIONS) {
+        if (!Utils::send_str_wrapper(connFd, "JasmineGraph server is busy. Please try again later.")) {
+            frontend_logger.error("Error writing to socket");
+        }
+        close(connFd);
+        currentFESession--;
+        return NULL;
+    }
+
     frontend_logger.info("Thread No: " + to_string(pthread_self()));
```

Comment:
Shall we remove the following two log lines? I do not see anypoint of having those here.

```
 frontend_logger.info("Thread No: " + to_string(pthread_self()));
    frontend_logger.info("Master IP: " + masterIP);
```

Reply:
Removed

Reply:
Ok